website_apps_store/controllers/main.py

```diff
@@ -23,7 +23,9 @@ def _get_search_domain(self, search, category, attrib_values):
         if search:
             for srch in search.split(" "):
                 domain += [
-                    '|', '|', '|', '|', '|', '|', '|', ('name', 'ilike', srch),
+                    '|', '|', '|', '|', '|', '|', '|', '|',
```

Comment:
Related to a security reason: Isn't it better to use expression.AND in this case since it santizes the 'srch' input (which comes from the internet)?

Reply:
@StephanRozendaal we have just overridden the core method and enhance it https://github.com/odoo/odoo/blob/11.0/addons/website_sale/controllers/main.py#L176